src/actions/getActiveSubscription.ts

```diff
@@ -10,10 +10,14 @@ type PricePlan = components['schemas']['PricePlan'];
 export default function getActiveSubscription(
   customerToken: string
 ): Promise<PricePlan | null> {
+  if (!customerToken) {
```

Comment:
I left this check to be sure `token` is passed (in case we want to expose this actions to public)